ui/packages/embed/src/Layout/index.tsx

```diff
@@ -11,6 +12,8 @@ type Props = {
 const Layout = ({ data }: Props) => {
   const [ready, setReady] = useState(false);
   const [loaded, setLoaded] = useState(0);
+  const { i18n } = useTranslation();
+  i18n.changeLanguage("en"); // make sure that embed is definitely using English localization, might not be necessary?
```

Comment:
one thing we could do is add a language property to the share url (this was my original plan). IE: https://gcsim.app/sh/123?lang=ja would force the site to be in jp (there is a built-in one for this already but don't remember what it should be) and then the embed request can also pick up the language and generate accordingly to allow for localized embeds.